arlas-rest/src/main/java/io/arlas/server/rest/collections/CollectionService.java

```diff
@@ -236,24 +236,27 @@ public Response put(
             @ApiParam(name = "pretty",
                     value = Documentation.FORM_PRETTY,
                     defaultValue = "false")
-            @QueryParam(value = "pretty") Boolean pretty
+            @QueryParam(value = "pretty") Boolean pretty,
+
+            @ApiParam(name = "isgeo", defaultValue = "true")
```

Comment:
i suggest to name the query param `checkgeo` (the same variable name)

Reply:
Fixed